lib/system/storage.toit

```diff
@@ -372,10 +372,23 @@ class Region extends ServiceResourceProxy:
         --to=to
         --max-size=max-size
 
+  /**
+  Deprecated. Use $(write --at bytes) instead.
+  */
   write --from/int bytes/ByteArray -> none:
     if not resource_: throw "ALREADY_CLOSED"
     flash-region-write_ resource_ from bytes
 
+  /**
+  Writes the given $data into the region at the given offset $at.
+
+  If the region has already data, it is bit-anded with the
```

Comment:
This isn't necessarily true. It depends on $erase-value, etc.

Reply:
done.